src/components/checkbox/index.tsx

```diff
@@ -122,7 +129,8 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     super(props);
 
     this.state = {
-      isChecked: new Animated.Value(this.props.value ? 1 : 0)
+      isChecked: new Animated.Value(this.props.value ? 1 : 0),
+      isValid: true
```

Comment:
If we want to support things like `validateOnStart` as we do elsewhere, this should be conditional on initial state.

Reply:
`validateOnStart` was not a requirement